app/main.py

```diff
@@ -157,39 +164,69 @@ async def summarize_visits():
             open(CSV_FILE_PATH, "w").close()
         except Exception as e:
             logger.error(f"Failed to write to the database: {e}")
-            raise HTTPException(status_code=500, detail=str(e))
+            return Response(status_code=500, content=str(e))
         else:
             logger.debug(f"Successfully wrote {len(minute_summary)} rows to the DB")
             # Clear the CSV file on successful write
             open(CSV_FILE_PATH, "w").close()
-        return Response(status_code=200)
+        return Response(
+            status_code=200,
+            content=f"Successfully summarized {len(df)} visits to {len(minute_summary)} per-minute entries",
+        )
 
 
-# Add a /summarize endpoint to trigger the summarize logic manually available only when developing
+@app.on_event("startup")
+@repeat_every(seconds=60 * 60 * 1)  # daily
```

Comment:
This is hourly currently?
```suggestion
@repeat_every(seconds=60 * 60 * 24)  # daily
```